README.md

```diff
@@ -1 +1,21 @@
 # Governance and Guidance
+
+This Repository hosts the collection of documents that define how the organization of Tauri-Apps works and understands itself. Unless these documents are clearly superceded in other repositories within this organization's collection, they are considered binding. 
+
+## Document List
+
+ORGANIZATION
+ - CODE OF CONDUCT
+ - SOCIAL CONTRACT
+ - WORKING GROUPS
```

Comment:
I assume will be overview and guidance. Since workgroups are otherwise autonomous and will need (the option) to document their own workings.